src/load.ts

```diff
@@ -7,12 +9,17 @@ import * as sops from './utils/sops'
 
 import { HydratedConfig } from './types'
 
-export const load = (): HydratedConfig => {
+export const load = (configDir = './config'): HydratedConfig => {
+  const normedConfigDir = path.normalize(configDir)
```

Comment:
nit: i think `normedConfigDir` is incorrect english here (at least i have never seen `normed` as a var name anywhere else so far)

```suggestion
  const normalizedConfigDir = path.normalize(configDir)
```

Reply:
updated